lib/hardware_services/quads_default_driver.py

```diff
@@ -0,0 +1,2 @@
+# this class will inherit from hardware_service.py and overwrite all of its methods
```

Comment:
+# This will be an abstract class (perhaps can use abc) that will serve as a generic interface
+# for QUADS to interact with its hardware.

+#   remove_host():
 +#   remove_cloud():
 +#   update_host():
 +#   update_cloud():
 +#   list_hosts():
 +#   list_clouds():

All this commands don't interact with hardware...